### PR TITLE
Font fixes.

### PR DESCRIPTION
**Pull request by camelburrito**

### CODE REVIEW

File: css/www/page-vars.css [hunk 1]
```diff
@@ -37,5 +37,5 @@
 
   --header-gradient: linear-gradient(to right, #280e75 0%, #066573 100%);
 
-  --font-family: Montserrat;
+  --font-family: Montserrat, -apple-system, BlinkMacSystemFont,"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
```
Comment (matharden) [inline css/www/page-vars.css]: Could do with a space after "BlinkMacSystemFont"

File: css/www/shared.css [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 h1,h2,h3,h4,h5,h6,
 .www-header {
-  font-family: Inconsolata;
+  font-family: Inconsolata,-apple-system, BlinkMacSystemFont,"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
```
Comment (matharden) [inline css/www/shared.css]: Space after "Inconsolata" and "BlinkMacSystemFont"

File: css/ampstart-base/base-page-vars.css [hunk 3]
```diff
@@ -48,11 +48,7 @@
   --color-input-primary: var(--color-font-accent);
   --color-input-font-secondary: var(--color-font-accent);
 
-  /* Never override this */
-  --base-font-family: -apple-system, BlinkMacSystemFont,"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
-
-  /* Override this */
-  --font-family: '';
+  --font-family: -apple-system, BlinkMacSystemFont,"Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", Arial, sans-serif;
```
Comment (matharden) [inline css/ampstart-base/base-page-vars.css]: Add space after "BlinkMacSystemFont".

File: css/www/page-vars.css (hunk 1 above)
Comment (camelburrito) [inline css/www/page-vars.css]: Done

File: css/www/shared.css (hunk 2 above)
Comment (camelburrito) [inline css/www/shared.css]: Done

File: css/ampstart-base/base-page-vars.css (hunk 3 above)
Comment (camelburrito) [inline css/ampstart-base/base-page-vars.css]: Done

**Pull request merged**